From 72fd09ac934f49ef1bdbd52d897f69d2b7fa5597 Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Mon, 8 Sep 2014 14:32:13 -0400
Subject: [scsi] hpsa: make tech preview info more verbose

Message-id: <1410186734-19986-83-git-send-email-thenzl@redhat.com>
Patchwork-id: 92095
O-Subject: [RHEL6.6 e-stor PATCH 82/83] hpsa: make tech preview info more verbose
Bugzilla: 1083556
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

bz#1083556

Added dev-id to the tech preview message
RH specific

Signed-off-by: Tomas Henzl <thenzl@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index de21a90..3efaf10 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -6221,6 +6221,7 @@ static int hpsa_lookup_board_id(struct pci_dev *pdev, u32 *board_id)
 {
  int i;
  u32 subsystem_vendor_id, subsystem_device_id;
+ char msg[80];
 
  subsystem_vendor_id = pdev->subsystem_vendor;
  subsystem_device_id = pdev->subsystem_device;
@@ -6239,7 +6240,8 @@ static int hpsa_lookup_board_id(struct pci_dev *pdev, u32 *board_id)
    return -ENODEV;
  }
 
- mark_tech_preview("hpsa: this controller", THIS_MODULE);
+ sprintf(msg, "hpsa: this controller (%#x)", subsystem_device_id);
+ mark_tech_preview(msg, THIS_MODULE);
 
  return ARRAY_SIZE(products) - 1; /* generic unknown smart array */
 }
-- 
1.7.1